src/main/java/org/violetmoon/quark/content/building/module/MoreStoneVariantsModule.java

```diff
@@ -65,13 +64,14 @@ public static void expandVanillaStone(ZRegister event, ZetaModule module, Block
 		NewStoneTypesModule.makeStone(event, module, raw, name, null, null, () -> true, null, ZetaBlock::new);
 	}
 	
-	private void add(ZRegister event, String name, MaterialColor color, SoundType sound, BooleanSupplier cond) {
+	private void add(ZRegister event, String name, MapColor color, SoundType sound, BooleanSupplier cond) {
 		add(event, name, color, sound, cond, ZetaBlock::new, ZetaPillarBlock::new);
 	}
 	
-	private void add(ZRegister event, String name, MaterialColor color, SoundType sound, BooleanSupplier cond, ZetaBlock.Constructor<ZetaBlock> constr, ZetaBlock.Constructor<ZetaPillarBlock> pillarConstr) {
-		Block.Properties props = Block.Properties.of(Material.STONE, color)
```

Comment:
Material.STONE => MapColor.STONE, NoteBlockInstrument.BASEDRUM

You forgot to assign the note block sound

Reply:
Whoops